### PR TITLE
Dist/21 december 2022

### PR DESCRIPTION
**Pull request by luxumbra**
Latest distro done. 

Always love seeing this on this repo. 😬 
image

### CODE REVIEW

**Comment by luxumbra** [comment]
@alalonde I've fixed the conflict I had in the ledger and the numbers still look correct, so am assuming all is good. ~~Am wondering if I should also make the seed deduct part of this PR?~~ I have also done the deduct of seeds from the ledger as part of this PR.

File: scripts/seed-minting-disburse.js
Comment (polimyl) [inline scripts/seed-minting-disburse.js:20]: dont the coordinape seeds also have to be included somehow?

Comment (alalonde) [reply scripts/seed-minting-disburse.js:20]: No, because they're not linked to SC at all

**Review by alalonde — APPROVED** [review]
lgtm

**Comment by luxumbra** [comment]
@alalonde I didn't merge right after your approval so ended up with another conflict on the ledger. Have fixed and pushed again - the conflict was due to JayFrank registering since I pushed but I've checked the order of entries and local doesnt throw any warnings so I think is good but a sanity check would be appreciated.

**Review by alalonde — COMMENTED** [review]
Thanks; ledger still looks fine

**Comment by luxumbra** [comment]
> Thanks; ledger still looks fine

Thanks.

**Pull request merged**
